core/src/com/unciv/ui/utils/ZoomableScrollPane.kt

```diff
@@ -366,4 +366,7 @@ open class ZoomableScrollPane(
     private fun onViewportChanged() {
         onViewportChangedListener?.invoke(maxX, maxY, getViewport())
     }
+
+    open fun checkX(deltaX: Float): Float = scrollX - deltaX
```

Comment:
These are not good names. What is the intent for these functions?

Reply:
These functions check if the coordinates are within the explored region

Reply:
So maybe getNewX? If a function is named "check" I expect to receive the result of a check, not the final information